docs/doxygen/doxyfile.in

```diff
@@ -2063,7 +2063,7 @@ MAN_LINKS              = NO
 # captures the structure of the code including all documentation.
 # The default value is: NO.
 
-GENERATE_XML           = YES
+GENERATE_XML           = NO
```

Comment:
I need this to generate docs for hse-project.github.io

Reply:
We could default the option docs=disabled?

Reply:
Oh good to know. I will add it back.